src/database/models/user.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const UserSchema = new mongoose.Schema({
+	id: { type: String },
+	registeredAt: { type: Number, default: Date.now() },
```

Comment:
This `Date.now()` will not do what you expect. This will calculate one date at startup and never change.

Instead use `Date.now` so this function can be called on creation!
https://mongoosejs.com/docs/defaults.html

Reply:
Seems that this is wrong on `guild.js` too

Reply:
I just double checked with FDM Monster and this is what I cover with tests there:
https://github.com/fdm-monster/fdm-monster/blob/fcc4707613521647508e4e5822535de887c7f41c/server/models/Auth/User.js#L19